app/assets/javascripts/application.js

```diff
@@ -255,7 +255,7 @@ function template_info(div, url) {
   // Use a post to avoid request URI too large issues with big forms
   $.ajax({
     type: "POST",
-    url: url + "&provisioning=" + build,
+    url: url + ((url.indexOf('?') == -1) ? '?' : '&') + "provisioning=" + build,
```

Comment:
I can't tell what advantage that provides over the current implementation.  Are you suggesting it needs to handle question marks in the path or host portions of the URL?